modules/seqexec/web/client/src/main/scala/seqexec/web/client/services/SeqexecWebClient.scala

```diff
@@ -33,17 +52,12 @@ object SeqexecWebClient extends ModelBooPicklers {
     Unpickle[A].fromBytes(ab)
   }
 
-  def sync(id: Observation.Id): Future[SequencesQueue[Observation.Id]] =
-    Ajax.get(
+  def sync(id: Observation.Id): Future[String] =
```

Comment:
This PR showed me we have an issue on the way requests are decoded. I'll work on that in another PR